src/__tests__/__snapshots__/relationships.test.ts.snap

```diff
@@ -80,6 +74,7 @@ exports[`Relationships createRelationshipsNode adds value when relationshipsList
         ],
       },
     ],
+    "length": 2,
```

Comment:
question: what is the length here?

Reply:
number of items in the columns. if you look up each has 2
```
        "values": [
          "test-source",
          "test-target",
        ],
```